File: src/main/java/com/saeed/tasky/models/dto/ProjectDto.java
package com.saeed.tasky.models.dto;

import com.saeed.tasky.models.Role;
import com.saeed.tasky.models.User;
import io.swagger.annotations.ApiModel;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Builder
@NoArgsConstructor
@AllArgsConstructor
@Data
@ApiModel(description = "All details about the customer. ")
public class ProjectDto {
    private User user;
    private Role role;
    private String name;
}

File: src/main/java/com/saeed/tasky/models/dto/ProjectDto.kt
package com.saeed.tasky.models.dto

import com.saeed.tasky.models.Role
import com.saeed.tasky.models.User

data class ProjectDto (
    var user: User? = null,
    var role: Role? = null,
    var projectName: String? = null,
    var projectIcon: String? = null,
    var projectId: Long? = null
)